packages/core/upload/admin/src/components/UploadAssetDialog/AddAssetStep/tests/FromComputerForm.test.js

```diff
@@ -5,11 +5,6 @@ import { TrackingProvider } from '@strapi/helper-plugin';
 import { IntlProvider } from 'react-intl';
 import { FromComputerForm } from '../FromComputerForm';
 
-jest.mock('axios', () => ({
```

Comment:
Do you have to mock getFetchClient instead?

Reply:
it works also without but yes, I can add the mock of the getFetchClient

Reply:
It works for sure. Does it throw an error in the console when running the test?

Reply:
I added the mock just to be sure it is all ok